.circleci/config.yml

```diff
@@ -0,0 +1,64 @@
+# These environment variables must be set in CircleCI UI
+#
+# AWS_ACCESS_KEY_ID
+# AWS_SECRET_ACCESS_KEY
+#
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: docker:18.02.0-ce
+    working_directory: ~/mozilla/mozilla-reports
+    steps:
+      - checkout
+      - run: python -m docere.cli render --knowledge-repo tests/data/kr
+
+  test:
+    machine:
+      enable: true
+    working_directory: ~/mozilla/mozilla-reports
+    steps:
+      - checkout
+      - run: sudo apt-get update
+      - run: sudo apt-get install python-dev
+      - run: sudo pip install git+ssh://git@github.com/airbnb/knowledge-repo.git
+      - run: insert some bash command to run tests
```

Comment:
With the switch to docere, there's no need for installing knowledge-repo at all, nor do I think there's any need for unit tests since docere has its own unit tests.

I'm imaging the following that should run for all pushes and PRs; it will only fail if docere throws a rendering error:

```
build:
  docker: python:3.7
    working_directory: /mozilla-reports
    steps:
      - checkout
      - run: pip install docere
      - run: mkdir /rendered
      - run: python -m docere.cli render --knowledge-repo . --output /rendered/post
      - persist_to_workspace:
          root: /rendered
          paths:
            - post/
```

And then only on master, run the upload:

```

  deploy:
    docker:
      - image: docker:18.02.0-ce
    working_directory: /mozilla-reports
    - steps:
      - attach_workspace:
        at: /rendered
      - run: upload /rendered/post to S3
```
```